libs/watt/src/lib/components/tabs/watt-link-tabs.component.ts

```diff
@@ -0,0 +1,100 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Component, ViewEncapsulation, contentChildren } from '@angular/core';
+import { MatTabsModule } from '@angular/material/tabs';
+import { RouterLink, RouterLinkActive, RouterOutlet } from '@angular/router';
+import { WattLinkTabComponent } from './watt-link-tab.component';
+
+@Component({
+  standalone: true,
+  selector: 'watt-link-tabs',
+  encapsulation: ViewEncapsulation.None,
+  imports: [MatTabsModule, RouterOutlet, RouterLink, RouterLinkActive],
+  styles: `
+    @use '@energinet-datahub/watt/utils' as watt;
+    .mat-mdc-tab-header {
+      box-shadow: var(--watt-bottom-box-shadow);
+      background-color: var(--watt-color-neutral-white);
+    }
+
+    .mat-mdc-tab-nav-panel {
+      display: block;
+      padding: var(--watt-space-ml);
+    }
+
+    .mat-mdc-tab-links,
+    .mat-mdc-focus-indicator.mat-mdc-tab-link.active .mdc-tab__text-label {
+      color: var(--mat-tab-header-active-label-text-color);
+    }
+
+    .mat-mdc-tab-labels,
+    .mat-mdc-tab-links {
+      padding-left: var(--watt-space-ml);
+    }
+
+    .mat-mdc-tab,
+    .mat-mdc-tab-link {
+      min-width: 120px;
```

Comment:
issue: for some reason the value here is not applied 🤔

![image](https://github.com/Energinet-DataHub/greenforce-frontend/assets/1096332/886c2625-5a15-4ac7-a0fb-06c1a9f68fe7)